.github/workflows/nightly-remove-stale-review-apps.yml

```
@@ -21,9 +21,42 @@ jobs:
+              direction: 'desc'
+            });
+            
+            const stalePRs = prs.data
+              .filter(pr => new Date(pr.updated_at) < thirtyDaysAgo)
+              .map(pr => pr.number);
+            
+            console.log('Found stale PRs:', stalePRs);
+            return stalePRs;
+
+      - name: Delete Stale Review Apps
+        uses: ./.github/actions/delete-control-plane-app
```

actionlint (1.7.4)

missing input "app_name" which is required by action "Delete Control Plane App" defined at "./.github/actions/delete-control-plane-app". all required inputs are "app_name", "org"

(action)